contracts/covenant/examples/schema.rs

```diff
@@ -0,0 +1,15 @@
+use cosmwasm_schema::{export_schema, remove_schemas, schema_for};
```

Comment:
Use the new schema? 
Example: https://github.com/Cosmos-Coin-Flip/Cosmos-Coin-Flip/blob/main/contracts/coin-flip/src/bin/schema.rs

Also need to add responses to queries to make it work, like this: 
https://github.com/Cosmos-Coin-Flip/Cosmos-Coin-Flip/blob/2442f84bac6747bff2d75c8a8905433cd6e55f43/contracts/coin-flip/src/msg.rs#L29

Reply:
ah yeah!